recipes/zospy/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "zospy" %}
+{% set version = "1.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/zospy-{{ version }}.tar.gz
+  sha256: 21f77d281b2f13b55af344340cddf8c93101af724924300b3ca04bad28f3552d
+
+build:
+  skip: True  # [not win]
```

This can be `noarch`
```suggestion
  noarch: python
```

---

I tried to use `noarch` before, but didn't manage to disable the Linux and MacOS builds. ZOSPy depends on Windows-specific libraries (e.g. winreg), which causes the import test to fail on other platforms. Because ZOSPy also depends on OpticStudio, which only supports Windows, it doesn't make sense to build ZOSPy for other platforms. How can I use `noarch` while skipping the builds on other platforms than Windows?